src/app/static/src/app/store/filteredData/getters.ts

```diff
@@ -104,26 +108,47 @@ export const getters = {
         for (const region in result) {
             const indicators = result[region];
             if (indicators.art) {
-                indicators.art.color = getColor(indicators.art, artRange, getters.colorFunctions.art);
+                indicators.art.color = getColor(indicators.art!.value, artRange.min, artRange.max, getters.colorFunctions.art);
             }
             if (indicators.prev) {
-                indicators.prev.color = getColor(indicators.prev, prevRange, getters.colorFunctions.prev);
+                indicators.prev.color = getColor(indicators.prev!.value, prevRange.min, prevRange.max, getters.colorFunctions.prev);
             }
         }
         return {
             indicators: result,
             artRange: artRange,
             prevRange: prevRange
         };
+    },
+    flattenedRegionOptions: function(state: FilteredDataState, getters: any, rootState: RootState, rootGetters: any) {
+        const option = getters.regionOptions;
+        return option ? flattenOption(option) : {};
+    },
+    flattenedSelectedRegionFilter: function(state: FilteredDataState, getters: any, rootState: RootState, rootGetters: any) {
+        return state.selectedChoroplethFilters.region ? flattenOption(state.selectedChoroplethFilters.region) : {};
+    },
+};
+
+const flattenOption = (regionFilter: NestedFilterOption) => {
+    let result = {} as any;
+    result[regionFilter.id] = regionFilter;
+    if (regionFilter.options) {
+        regionFilter.options.forEach(o =>
+            result = {
+                ...result,
+                ...flattenOption(o as NestedFilterOption)
+            });
     }
+    return result;
 };
 
-const getColor = (data: IndicatorValues, range: IndicatorRange, colorFunction: (t: number) => string) => {
-    let rangeNum = (range.max  && (range.max != range.min)) ? //Avoid dividing by zero if only one value...
-                    range.max - (range.min || 0) :
+export const getColor = (value: number, min: number | null, max: number | null, colorFunction: (t: number) => string) => {
+    min = (min || 0);
+    let rangeNum = (max  && (max != min)) ? //Avoid dividing by zero if only one value...
+                    max - min :
                     1;
 
-    const colorValue = data!.value / rangeNum;
+    const colorValue = (value - min) / rangeNum;
```

Comment:
I think this was a bug - color value should map min -> max to 0 -> 1, so need to subtract min from the value. Usually min will be close to 0 which is probably why we didn't notice this!